### PR TITLE
Create Table_with_string

### PR DESCRIPTION
**Pull request by vijayanuthalapatii**
Creating a table with the index letter as answer from the string

### CODE REVIEW

File: vijaya/Table_with_string [hunk 1]
```diff
@@ -0,0 +1,17 @@
+s = "Hi Team, the question is given by me & it should be optimizable code & the time complexity & space complexity should be equal or less than O(n)."
+t = 2
+s1 = ""
+for i in s:
+    if (ord(i) >= 97 and ord(i) <= 122) or (ord(i) >= 65 and ord(i) <= 90):
+        s1 += i
+print(s1)
+#  to find the length of the given string
+length = 0
+for j in s1:
```
Comment (Besquarelibrary) [inline vijaya/Table_with_string:11]: why this loop here?

File: vijaya/Table_with_string [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    length += 1
+print(length)
+#
+for i in range(1, length+1):
```
Comment (Besquarelibrary) [inline vijaya/Table_with_string:16]: what is the significance of this loop?

**Review by Besquarelibrary — COMMENTED** [review]
Can you comment what is these loops logic?

File: vijaya/Table_with_string (hunk 1 above)
Comment (vijayanuthalapatii) [reply vijaya/Table_with_string:11]: As I mentioned in the code as comment , that loop is to find the length of the string

File: vijaya/Table_with_string (hunk 2 above)
Comment (vijayanuthalapatii) [reply vijaya/Table_with_string:16]: This loop will start the range from 1 to last index to multiply with n

**Review by Besquarelibrary — APPROVED** [review]
looks good

**Pull request merged**